lib/mix/lib/mix/tasks/compile.app.ex

```diff
@@ -17,11 +17,11 @@ defmodule Mix.Tasks.Compile.App do
 
   The most commonly used options are:
 
-    * `:extra_applications` - a list of Erlang/Elixir applications
+    * `:extra_applications` - a list of OTP/Elixir applications
```

Comment:
I think we should either say Erlang/Elixir or OTP applications but not "OTP/Elixir" applications.

Reply:
Let's go with "OTP applications" because in the next sentence, it mentions "For example, here you can declare a dependency on applications that ship with Erlang/OTP or Elixir"